crates/router/src/connector/dummyconnector.rs

```diff
@@ -136,10 +138,26 @@ impl ConnectorIntegration<api::Authorize, types::PaymentsAuthorizeData, types::P
 
     fn get_url(
         &self,
-        _req: &types::PaymentsAuthorizeRouterData,
-        _connectors: &settings::Connectors,
+        req: &types::PaymentsAuthorizeRouterData,
+        connectors: &settings::Connectors,
     ) -> CustomResult<String, errors::ConnectorError> {
-        Err(errors::ConnectorError::NotImplemented("get_url method".to_string()).into())
+        let payment_method_data = req.request.payment_method_data.clone();
+        let payment_method_type = req
+            .to_owned()
+            .request
+            .payment_method_type
+            .ok_or_else(connector_utils::missing_field_err("payment_method_type"))?;
```

Comment:
```suggestion
        let payment_method_type = req
            .to_owned()
            .request
            .get_payment_method_type()?;
```

Add `get_payment_method_type()` in PaymentsAuthorizeRequestData in connector/utils.rs